doc/api/fs.md

```diff
@@ -7510,6 +7510,17 @@ added:
 
 Free blocks available to unprivileged users.
 
+Example:
+```js
+  const fs = require('fs');
+  // Calculate available space in bytes
+  fs.statfs('.', (err,stats) => {
+  const availableSpace = stats.bavail * stats.bsize;
+
+  console.log(`Available space for unprivileged users: ${availableSpace} bytes`);
+});
+```
```

Comment:
1. Can you include both a cjs and mjs example?
2. Can you fix the indent


Ditto on all other examples